<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="9069" started-at="2013-04-27T13:24:26Z" finished-at="2013-04-27T13:24:35Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="9069" started-at="2013-04-27T13:24:26Z" finished-at="2013-04-27T13:24:35Z">
      <class name="com.example.tests.GroupCreationTests">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.GroupCreationTests@4a13fc0f]" name="setUp" is-config="true" duration-ms="16" started-at="2013-04-27T14:24:26Z" finished-at="2013-04-27T14:24:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testGroupCreationWithValidData(com.example.tests.GroupData)[pri:0, instance:com.example.tests.GroupCreationTests@4a13fc0f]" name="testGroupCreationWithValidData" duration-ms="8631" started-at="2013-04-27T14:24:26Z" data-provider="groupsFromFile" finished-at="2013-04-27T14:24:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GroupData [name=test2011679902, header=, footer=test815756619 test test]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unsupported browser: firefox]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unsupported browser: firefox
	at com.example.fw.ApplicationManager.getDriver(ApplicationManager.java:77)
	at com.example.fw.WebDriverHelperBase.<init>(WebDriverHelperBase.java:16)
	at com.example.fw.NavigationHelper.<init>(NavigationHelper.java:8)
	at com.example.fw.ApplicationManager.navigateTo(ApplicationManager.java:33)
	at com.example.fw.GroupHelper.createGroup(GroupHelper.java:46)
	at com.example.tests.GroupCreationTests.testGroupCreationWithValidData(GroupCreationTests.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGroupCreationWithValidData -->
        <test-method status="FAIL" signature="testGroupCreationWithValidData(com.example.tests.GroupData)[pri:0, instance:com.example.tests.GroupCreationTests@4a13fc0f]" name="testGroupCreationWithValidData" duration-ms="1" started-at="2013-04-27T14:24:35Z" data-provider="groupsFromFile" finished-at="2013-04-27T14:24:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GroupData [name=test614751556, header=, footer=]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unsupported browser: firefox]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unsupported browser: firefox
	at com.example.fw.ApplicationManager.getDriver(ApplicationManager.java:77)
	at com.example.fw.WebDriverHelperBase.<init>(WebDriverHelperBase.java:16)
	at com.example.fw.NavigationHelper.<init>(NavigationHelper.java:8)
	at com.example.fw.ApplicationManager.navigateTo(ApplicationManager.java:33)
	at com.example.fw.GroupHelper.createGroup(GroupHelper.java:46)
	at com.example.tests.GroupCreationTests.testGroupCreationWithValidData(GroupCreationTests.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGroupCreationWithValidData -->
        <test-method status="FAIL" signature="testGroupCreationWithValidData(com.example.tests.GroupData)[pri:0, instance:com.example.tests.GroupCreationTests@4a13fc0f]" name="testGroupCreationWithValidData" duration-ms="31" started-at="2013-04-27T14:24:35Z" data-provider="groupsFromFile" finished-at="2013-04-27T14:24:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GroupData [name=, header=test843008632, footer=]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unsupported browser: firefox]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unsupported browser: firefox
	at com.example.fw.ApplicationManager.getDriver(ApplicationManager.java:77)
	at com.example.fw.WebDriverHelperBase.<init>(WebDriverHelperBase.java:16)
	at com.example.fw.NavigationHelper.<init>(NavigationHelper.java:8)
	at com.example.fw.ApplicationManager.navigateTo(ApplicationManager.java:33)
	at com.example.fw.GroupHelper.createGroup(GroupHelper.java:46)
	at com.example.tests.GroupCreationTests.testGroupCreationWithValidData(GroupCreationTests.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGroupCreationWithValidData -->
        <test-method status="FAIL" signature="testGroupCreationWithValidData(com.example.tests.GroupData)[pri:0, instance:com.example.tests.GroupCreationTests@4a13fc0f]" name="testGroupCreationWithValidData" duration-ms="31" started-at="2013-04-27T14:24:35Z" data-provider="groupsFromFile" finished-at="2013-04-27T14:24:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[GroupData [name=test-729176005, header=, footer=test-2121173268]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unsupported browser: firefox]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unsupported browser: firefox
	at com.example.fw.ApplicationManager.getDriver(ApplicationManager.java:77)
	at com.example.fw.WebDriverHelperBase.<init>(WebDriverHelperBase.java:16)
	at com.example.fw.NavigationHelper.<init>(NavigationHelper.java:8)
	at com.example.fw.ApplicationManager.navigateTo(ApplicationManager.java:33)
	at com.example.fw.GroupHelper.createGroup(GroupHelper.java:46)
	at com.example.tests.GroupCreationTests.testGroupCreationWithValidData(GroupCreationTests.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGroupCreationWithValidData -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.GroupCreationTests@4a13fc0f]" name="tearDown" is-config="true" duration-ms="172" started-at="2013-04-27T14:24:35Z" finished-at="2013-04-27T14:24:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.GroupCreationTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
